testeng/resources/cleanup-python-code-create-pr.sh

```diff
@@ -3,6 +3,18 @@ set -eu -o pipefail
 
 # Jenkins won't set an environment variable if it's empty.
 # Set this to an empty string if it's unset.
+
+command -v nvm >/dev/null 2>&1 || {
+  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash;
+  source ~/.bashrc;
+  source ~/.nvm/nvm.sh;
+  }
+
+nvm install 16
+nvm use 16
+
+node --version
+
 PACKAGES="${PACKAGES:-}"
```

Comment:
You have placed your code after an explanatory comment for the `PACKAGES="${PACKAGES:-}"` line. So it would be best if you moved your addition up or down to not separate the line from its explanation.